Autopilot/Path Manager/MPL3115A2.c

```diff
@@ -30,36 +33,80 @@ char initAltimeter() {
 #endif
     }
     return altimeterConnected;
+    // set pressure 
 }
 
-void calibrateAltimeter(long int pressure){
-    int msb = pressure >> 9;
-    int lsb = (pressure >> 1)& 255;
-    int offset = 0;
-    sendMessage(I2C_SLAVE_ADDRESS, BAR_IN_MSB, (char*)&msb, 1, WRITE);
-    sendMessage(I2C_SLAVE_ADDRESS, BAR_IN_LSB, (char*)&lsb, 1, WRITE);
-    sendMessage(I2C_SLAVE_ADDRESS, OFF_H, (char*)&offset, 1, WRITE);
+void calibrateAltimeter(int altitudeOffset){
```

Comment:
Now that i think of it, you can avoid having to do the error check if you just make the parameter type a `int8_t`, or an 8-bit signed integer